### PR TITLE
les-427 add categories for user

### PR DESCRIPTION
**Pull request by valer-sky**

### CODE REVIEW

File: src/components/UserCatigories.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+type UserCategoriesProps = {
+  // id: number; use only for admin
+  imagePreview: string;
+  name: string;
+  text: string;
+  bgColor: "blue" | "pink" |"grey"| "orange";
+};
+
+const UserCategories: React.FC<UserCategoriesProps> = (props) => {
```
Comment (Terapool) [inline src/components/UserCatigories.tsx:46]: Please, rename to singular form: UserCategory, or CategoryCard

File: src/components/body/content/categories/index.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.wrapper__categories {
+    width: 1376px;
+    height: 828px;
+}
+
+.categories {
+    align-items: flex-start;
```
Comment (Terapool) [inline src/components/body/content/categories/index.scss:7]: Do you use here grid or flex? It looks like a mess...

align-items: flex-start;
...
display: grid;

**Review by Terapool — CHANGES_REQUESTED** [review]

File: src/components/UserCatigories.tsx (hunk 1 above)
Comment (valer-sky) [reply src/components/UserCatigories.tsx:46]: renamed

File: src/components/body/content/categories/index.scss (hunk 2 above)
Comment (valer-sky) [reply src/components/body/content/categories/index.scss:7]: this using grid)

**Review by DimitarShemet — APPROVED** [review]

**Review by Terapool — APPROVED** [review]

**Review by OxygenO22 — APPROVED** [review]

**Pull request merged**